### PR TITLE
Reference:#1gb36z3, Improvement: Adding Same day to woocommerce

### PR DESCRIPTION
**Pull request by davidmaring7**
# Reference

- Task [#1gb36z3](https://app.clickup.com/t/1gb36z3)

# Why I'm adding these lines of code

- To add same day to woocommerce

# Test

Captura de Pantalla 2021-09-15 a la(s) 8 42 36 a  m

# Message to the "nuevasfuncionalidades" channel

- (Mejora) [#1gb36z3](https://app.clickup.com/t/1gb36z3) Adaptar plugin woocommerce para recibir estructura SDD

### CODE REVIEW

File: shipit/trunk/readme.txt [hunk 1]
```diff
@@ -54,8 +54,11 @@ Sí, con las acciones en lote, más información en el siguiente link: https://s
 
 == Changelog ==
 
+= 3.5.0 =
+* Agrega la funcionalidad para aceptar envios con entrega el mismo dia.
+
 = 3.3.5 =
-* Fix order bulk action
+* Arregla problema de creacion de envios en accion masiva
```
Comment (multinauta) [inline shipit/trunk/readme.txt:61]:
```suggestion
* Arregla problema de creación de envíos en acción masiva
```

File: shipit/trunk/src/shipit_service/rate.php [hunk 2]
```diff
@@ -98,26 +98,9 @@ function setParcel($parcel = array(), $cart = false) {
       $this->cart = $cart;
     }
 
-    function getRateDescription($checkout, $days, $woocommerceSetting) {
+    function getRateDescription($checkout, $drescription, $woocommerceSetting) {
```
Comment (multinauta) [inline shipit/trunk/src/shipit_service/rate.php:101]:
```suggestion
    function getRateDescription($checkout, $description, $woocommerceSetting) {
```

File: shipit/trunk/readme.txt [hunk 3]
```diff
@@ -54,8 +54,11 @@ Sí, con las acciones en lote, más información en el siguiente link: https://s
 
 == Changelog ==
 
+= 3.5.0 =
+* Agrega la funcionalidad para aceptar envios con entrega el mismo dia.
```
Comment (gcamposm) [inline shipit/trunk/readme.txt:58]:
```suggestion
* Agrega la funcionalidad para aceptar envíos con entrega el mismo día.
```

File: shipit/trunk/readme.txt (hunk 1 above)
Comment (gcamposm) [inline shipit/trunk/readme.txt:61]: Maybe 'Corrige/Repara'
```suggestion
* Arregla problema de creación de envíos en acción masiva
```

File: shipit/trunk/src/shipit_service/rate.php (hunk 2 above)
Comment (gcamposm) [inline shipit/trunk/src/shipit_service/rate.php:101]:
```suggestion
    function getRateDescription($checkout, $description, $woocommerceSetting) {
```

**Review by multinauta — CHANGES_REQUESTED** [review]
- in the tag folder, it must not be compressed
- in the test section you could place the checkout image working

File: shipit/trunk/readme.txt (hunk 3 above)
Comment (davidmaring7) [reply shipit/trunk/readme.txt]: Done, thanks @gcamposm

File: shipit/trunk/readme.txt (hunk 1 above)
Comment (davidmaring7) [reply shipit/trunk/readme.txt]: Done @multinauta thanks👍🏻

File: shipit/trunk/readme.txt [hunk 4]
```diff
@@ -54,8 +54,11 @@ Sí, con las acciones en lote, más información en el siguiente link: https://s
 
 == Changelog ==
 
+= 3.5.0 =
+* Agrega la funcionalidad para aceptar envíos con entrega el mismo día.
+
 = 3.3.5 =
-* Fix order bulk action
+* Arregla problema de creación de envíos en acción masiva
```
Comment (gcamposm) [inline shipit/trunk/readme.txt:61]: Maybe "Corrige" / "Repara"

**Review by gcamposm — APPROVED** [review]

**Review by anyelopetit — APPROVED** [review]

**Review by multinauta — APPROVED** [review]

**Review by HaNdlezz — APPROVED** [review]

**Review by hiro-riveros — APPROVED** [review]

**Pull request merged**